app/assets/javascripts/app/views/comment_stream_view.js

```diff
@@ -22,7 +22,6 @@ app.views.CommentStream = app.views.Base.extend({
   setupBindings: function() {
     this.model.comments.bind('add', this.appendComment, this);
     this.model.bind("commentsExpanded", this.storeTextareaValue, this);
```

Comment:
Since we don't re-render the whole stream anymore it looks like you can remove this, remove the `storeTextareaValue` method and also all occurrences of the stored textarea value. There is a spec that says "refills the comment textbox on success". I guess the spec should still work but the description would have to be reworded.